src/server/views/prototypes/site-review/reinspection.njk

```diff
@@ -0,0 +1,48 @@
+{% extends 'layouts/default.layout.njk' %}
+
+{% set pageTitle = 'Premises' %}
+
+{% block content %}
+
+
+          <h4 class="heading-medium">Did you do a reinspection?</h4>
+          <form action="form">
+          <div class="form-group" style="margin-top: 5px;">
+            <fieldset>
+                <div class="multiple-choice" data-target="VRM">
+                    <input id="r-radio-1" name="radio-inline-group1" type="radio" value="Yes" aria-controls="VRM" aria-expanded="false"> <label for="r-radio-1">Yes</label>
+                </div>
+                <div class="panel panel-border-narrow js-hidden" id="VRM" aria-hidden="true">
+                    <label class="form-label" for="VRM">Registration mark of the reinspected vehicle</label>
+                    <input class="form-control" name="VRM" type="text" id="VRM">
+                </div>
+                <div class="multiple-choice" data-target="reinspection-blocked">
+                <input id="r-radio-2" name="radio-inline-group1" type="radio" value="No" aria-controls="reinspection-blocked" aria-expanded="false"> <label for="r-radio-2">No</label>
+                </div>
+                <div class="form-group" style="margin-top: 5px; margin-bottom: 0px;">
```

Comment:
You should not use inline styles.

Reply:
No worries, I didn't realise as we did this on the last framework

Reply:
Thanks, sorry its not valid. You can either find a class that suits by looking at the scss files or add your own, which can be then reviewed.

Example:
```
<div class="form-group u-margin-top-5">
...
</div>
```

File: https://github.com/dvsa/front-end/blob/master/src/assets/scss/modules/utils/partials/_u-margin.scss